util/environment.ts

```diff
@@ -0,0 +1,54 @@
+const DEFAULT_ENVIRONMENTS_REGEX: Record<EnvironmentName, string> = {
+  local: '^(:?localhost:\\d{2,5}|(?:127|192)(?:\\.[0-9]{1,3}){3})',
+  pr: '^cowfi-git-[\\w\\d-]+cowswap\\.vercel\\.app',
+  development: '^(develop.cow.fi|swap-develop.vercel.app)',
+  production: '^cow.fi$',
+}
+
+function getRegex(env: EnvironmentName) {
+  const regex = process.env[`REACT_APP_DOMAIN_REGEX_${env.toUpperCase()}`] || DEFAULT_ENVIRONMENTS_REGEX[env]
```

Comment:
Given the problem you recently solved with dynamic env vars with the RPC url, could this cause the same problem?

Reply:
Good point!

Reply:
Fixed!